### PR TITLE
Lab/remove unused icons

### PR DESCRIPTION
**Pull request by juanedi**
This PR removes some icons that are now inlined in the monolith.

Replaced use cases can be found here: https://github.com/NoRedInk/NoRedInk/pull/23994
Part of https://www.pivotaltracker.com/story/show/166191815

### CODE REVIEW

File: src/Nri/Ui/Button/V6.elm [hunk 1]
```diff
@@ -63,7 +63,7 @@ import Nri.Ui.AssetPath as AssetPath exposing (Asset)
 import Nri.Ui.Colors.Extra as ColorsExtra
 import Nri.Ui.Colors.V1 as Colors
 import Nri.Ui.Fonts.V1
-import Nri.Ui.Icon.V4 as Icon exposing (IconType)
+import Nri.Ui.Icon.V5 as Icon exposing (IconType)
```
Comment (tesk9) [inline src/Nri/Ui/Button/V6.elm]: Ooo wait we can't do this, can we? Won't this change will break all our existing uses of Nri.Ui.Button.V6?

I ran `elm diff`, and it says this is fine, which seems suspicious to me. I'm going to investigate a bit more.

Comment (tesk9) [reply src/Nri/Ui/Button/V6.elm]: Yeah it looks like it's this: https://github.com/elm/compiler/issues/1777

Comment (juanedi) [reply src/Nri/Ui/Button/V6.elm]: Ouch, good catch! I'll change that back to how it was before.

**Comment by tesk9** [comment]
@juanedi I deleted a bunch more stuff! how does this look to you?

File: src/Nri/Ui/Icon/V5.elm [hunk 2]
```diff
@@ -1,173 +1,109 @@
...
-@docs assignmentTypeQuickWrite
-@docs assignmentTypeQuiz
-@docs assignmentTypeSelfReview
-@docs assignmentTypeWritingCycle
-@docs attention
-@docs bang
-@docs bulb
-@docs calendar
-@docs caret
+
```
Comment (juanedi) [inline src/Nri/Ui/Icon/V5.elm]: 😻

File: src/Nri/Ui/Button/V6.elm [hunk 3]
```diff
@@ -63,7 +63,7 @@ import Nri.Ui.AssetPath as AssetPath exposing (Asset)
 import Nri.Ui.Colors.Extra as ColorsExtra
 import Nri.Ui.Colors.V1 as Colors
 import Nri.Ui.Fonts.V1
-import Nri.Ui.Icon.V5 as Icon exposing (IconType)
```
Comment (juanedi) [inline src/Nri/Ui/Button/V6.elm]: Thank you for fixing this!

**Comment by juanedi** [comment]
This looks great to me! 🎉
Should we merge?

**Pull request merged**